index.html

```diff
@@ -75,22 +77,24 @@
           <div class="box">
             <div class="field is-grouped">
               <p class="control is-expanded">
-                <input class="input">
+                <label hidden for="notify-me-input">get news</label>
+                <input class="input" id="notify-me-input">
               </p>
               <p class="control">
-                <a class="button is-info">Notify Me</a>
+                <button class="button is-link">Notify Me</button>
               </p>
             </div>
           </div>
         </div>
       </div>
     </div>
-  </div>
+  </header>
 
```

Comment:
I suggest adding "bread crumbs" section:

```suggestion
  <div id="breadcrumbs">
    <nav class="breadcrumb is-centered is-medium" role="breadcrumb">
      <ul>
        <li><a href="./" class="current" aria-current="page">MainPage</a><span></span></li>
        <li><a href="index.html#nav">Pictures</a><span></span></li>
        <li><a href="index.html#services-section">Services</a></li>
      </ul>
    </nav>
  </div>
  
  === styles.scc == 
  .breadcrumb {
  display: flex;
  padding: 20px;
}

.breadcrumb a {
  color: #1b4fa1;
}

.breadcrumb a:hover {
  text-decoration: underline;
}

.breadcrumb a.current {
  text-decoration: none;
  pointer-events: none;
  color: #000000;
}
```